frontend/src/components/mine/Permit/MinePermitInfo.js

```diff
@@ -107,49 +99,54 @@ const getPermittees = (partyRelationships, permits) =>
 const getPermitteeName = (permittees) =>
   permittees[0] ? permittees[0].party.party_name : Strings.EMPTY_FIELD;
 
-const transformRowData = (permits, partyRelationships) => {
-  const latest = permits[0];
-  const first = permits[permits.length - 1];
+const transformRowData = (permit, partyRelationships) => {
+  const latestAmendment = permit.amendments[0];
+  const firstAmendment = permit.amendments[permit.amendments.length - 1];
 
-  const permittees = getPermittees(partyRelationships, permits);
+  const permittees = getPermittees(partyRelationships, permit);
   const permitteeName =
     partyRelationships.length === 0 ? Strings.LOADING : getPermitteeName(permittees);
 
   return {
-    key: latest.permit_guid,
-    lastAmended: formatDate(latest.issue_date),
-    permitNo: latest.permit_no || Strings.EMPTY_FIELD,
-    firstIssued: formatDate(first.issue_date) || Strings.EMPTY_FIELD,
+    key: permit.permit_guid,
+    lastAmended: (latestAmendment && formatDate(latestAmendment.issue_date)) || Strings.EMPTY_FIELD,
```

Comment:
It looks like `.issue_date` can be `null`?
https://mds-schemaspy-538-empr-mds-dev.pathfinder.gov.bc.ca/tables/permit_amendment.html

We should probably check for that before passing it into the format function

Reply:
Done. formatDate does handle nulls, but it renders as an empty string instead of 'N/A', so either way I've now updated it to check for null and render Strings.EMPTY_FIELD.

Reply:
Perfect! Thanks 👍